src/real.rs

```diff
@@ -777,6 +828,7 @@ pub trait Real: Num + Copy + NumCast + PartialOrd + Neg<Output = Self> {
     fn atanh(self) -> Self;
 }
 
+#[cfg(any(feature = "std", feature = "libm"))]
 impl<T: Float> Real for T {
```

Comment:
We need to guard the trait definition in the same way. Features must additive, but adding a blanket impl to a trait is a breaking change. That's not an issue if the trait never exists without the blanket impl.

Reply:
This is now normally fixed [in this commit](https://github.com/rust-num/num-traits/pull/99/commits/583f2c71a3ff023de0b5d12714c95d21de5c9f6e).